src/pages/item/item.html

```diff
@@ -15,28 +15,28 @@
       <ion-item>
 				<ion-label floating color="primary">Brand</ion-label>
 				<ion-select [(ngModel)]="item.brandID" name="brand">
-          <ion-option *ngFor="let brand of brands" value="{{brand.id}}">{{brand.name}}</ion-option>
+          <ion-option *ngFor="let brand of brands" value="{{brand.brandID}}">{{brand.name}}</ion-option>
```

Comment:
Shouldn't `value` be bound here instead of using `{{this}}` notation?

(same for other properties in this file)